keras_tuner/engine/hypermodel.py

```diff
@@ -80,6 +80,22 @@ def _build_wrapper(self, hp, *args, **kwargs):
             hp = hp.copy()
         return self._build(hp, *args, **kwargs)
 
+    def fit(self, hp, model, *args, **kwargs):
+        """Train the model.
+
+        Args:
+            hp: HyperParameters.
+            model: `keras.Model` built in the `build()` function.
+            **kwargs: Anything the user defines. They are passed from
+                `Tuner.search()`. if contains `callbacks`, a list of prebuild
+                Keras callbacks for model checkpointing and tensorboard
+                configuration will be added to callbacks provided by the user.
+
+        Returns:
+            A `History` object, a similar dictionary, or a single value.
```

Comment:
Please provide more details about the dict and single value cases. Also mention that `History` instance is the output of `model.fit()`